internal/controller/metrics/metrics.go

```diff
@@ -0,0 +1,25 @@
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"sigs.k8s.io/controller-runtime/pkg/metrics"
+)
+
+var (
+	accessRequestStatusTotal = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "access_request_status_total",
+			Help: "Total number of AccessRequests by status",
+		},
+		[]string{"accessRequestStatus"},
+	)
+)
+
+func init() {
+	metrics.Registry.MustRegister(accessRequestStatusTotal)
+}
+
+// RecordAccessRequestStatus increments the counter for a given AccessRequest status
+func RecordAccessRequestStatus(status string) {
```

Comment:
nit: Just to keep things standardized, please rename this function to `IncrementAccessRequestCounter` or  `AccessRequestCounterInc`